README.md

```diff
@@ -136,27 +199,27 @@ E.g.
 
 ```bash
 Generate a MeasureReport by calculating a measure on a patient bundle:
-  - fqm-execution reports -m /path/to/measure/bundle.json -p /path/to/patient/bundle.json > reports.json
+  - fqm-execution reports -m /path/to/measure/bundle.json -p /path/to/patient/bundle.json  -o reports.json
 
 Generate a MeasureReport by calculating a measure on multiple patient bundles:
-  - fqm-execution reports -m /path/to/measure/bundle.json -p /path/to/patient1/bundle.json /path/to/patient2/bundle.json > reports.json
+  - fqm-execution reports -m /path/to/measure/bundle.json -p /path/to/patient1/bundle.json /path/to/patient2/bundle.json -o reports.json
 ```
 
 ### ValueSets
 
-If the Measure bundle provided doesn't contain all the required `ValueSet` resources (with expansions or composes) to calculate the measure, an API key can be provided to resolve the valuesets from their provided URLs. Currently only tested with valuesets from [The NLM FHIR Valueset API](https://cts.nlm.nih.gov/fhir).
+If the Measure bundle provided doesn't contain all the required `ValueSet` resources (with expansions or composes) to calculate the measure, an API key can be provided to resolve the valuesets from their provided URLs. Currently only tested with ValueSets from [The NLM FHIR Valueset API](https://cts.nlm.nih.gov/fhir).
 
 To find your VSAC API key, sign into [the UTS homepage](https://uts.nlm.nih.gov/uts/), click on `My Profile` in the top right, and copy the `API KEY` value from the `UMLS Licensee Profile`.
 
 ### TypeScript
 
-`fqm-execution` exports custom-defined TypeScript interfaces used within the code to allow for easy integration into other TypeScript projects. The TypeScript files defining these interfaces can be found [here](https://github.com/projecttacoma/fqm-execution/tree/master/src/types)
+`fqm-execution` exports custom-defined TypeScript interfaces used within the code to allow for easy integration into other TypeScript projects. The TypeScript files defining these interfaces can be found [here](https://github.com/projecttacoma/fqm-execution/tree/master/src/types).
 
 ## Local Development
 
 ### Prerequisites
 
-- [Node.js >=10.15.1](https://nodejs.org/en/)
+- [Node.js >=14.0.0](https://nodejs.org/en/)
 - [Git](https://git-scm.com/)
 
 ### Local Installation/Usage
```

Comment:
When I click "Display the rich diff" in the upper right corner, it seems like the link for this section in the table of contents is not working :

![image](https://user-images.githubusercontent.com/30158156/202537227-bb40c846-7045-490a-b0a8-5535c67b2b21.png)

Reply:
Looks like github markdown changed in some way.. actual link appears to be `#local-installationusage` now, which I guess we should update to?

Reply:
Updated it to `#local-installationusage`, let me know if it still doesn't work!

Reply:
Looks like this works now!